docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx

```diff
@@ -117,6 +119,16 @@ $ echo <Var name="version-server-url/path" />/<Var name="release-channel" /> | s
 Make sure not to include `https://` as a prefix to the server address.
 </Admonition>
 
+If the agent was not installed using the system package manager, follow the instructions on the
```

Comment:
Can we just embed the same instructions here instead of redirecting users to another page? Move it into a partial if needed and include in both places.